lib/active_model/serializer/configuration.rb

```diff
@@ -19,6 +19,7 @@ def config.array_serializer
           collection_serializer
         end
 
+        config.default_include_tree = '*'
```

Comment:
> How about default_nesting_level?

- `default_included_associations`? 
- `default_included_relationships`?
- `default_includes`?
- `default_included_resources`?

ref: 

- https://github.com/rails-api/active_model_serializers/blob/v0.5.0/lib/active_model/serializer.rb#L402-L424
- https://github.com/rails-api/active_model_serializers/blob/v0.7.0/lib/active_model/serializer.rb#L287-L297
- https://github.com/rails-api/active_model_serializers/blob/v0.8.0/lib/active_model/serializer.rb#L364-L369
- https://github.com/rails-api/active_model_serializers/blob/1e0c9ef93b96c640381575dcd30be07ac946818b/lib/active_model/serializer.rb#L240-L245

> A per-serializer config would raise issues (for instance parent has '**' and child has '*').

well, if the AuthorSerializer has a default includes of 'posts.comments' (which is crazy), then it would presumably set the includes of the PostSerializer to 'comments' and the includes of CommentSerializer to empty, right?